nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -117,15 +142,15 @@ def create_nncf_graph(model: torch.fx.GraphModule) -> PTNNCFGraph:
         """
 
         nncf_graph = PTNNCFGraph()
-
+        is_shared_const = GraphConverter._check_shared_constants()
```

Comment:
```suggestion
        const_targets = {}
```

Reply:
Regarding this, there are 2 assumptions we can make, which one is more suitable?
1. The transformation to remove and reconnect shared constants has already been called before calling the GraphConverter.
2. The model has not been transformed to join the shared constants together. 
in each of these scenarios, the way to handle `is_shared` attribute will be different. With the first one being just being a simple `is_shared_node = len(source_node.users) > 1 if source_node.op in ("get_attr",) else False`